ovirt/resource_ovirt_vm.go

```diff
@@ -92,6 +92,41 @@ var vmSchema = map[string]*schema.Schema{
 			Type: schema.TypeString,
 		},
 	},
+	"template_disk_attachment_override": {
+		Type:        schema.TypeSet,
+		Optional:    true,
+		ForceNew:    true,
+		Description: "Override parameters for disks obtained from templates.",
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"disk_id": {
+					Type:             schema.TypeString,
+					Required:         true,
+					ForceNew:         true,
+					Description:      "ID of the disk to be changed.",
+					ValidateDiagFunc: validateUUID,
+				},
+				"format": {
+					Type:        schema.TypeString,
+					Optional:    true,
+					ForceNew:    true,
+					Description: "Disk format for the override. Can be 'raw' or 'cow'.",
+					ValidateDiagFunc: validateEnum(
```

Comment:
I think we can reuse [validateFormat](https://github.com/oVirt/terraform-provider-ovirt/blob/main/ovirt/validation.go#L66-L90) here